glamor/glamor_gradient.c

```diff
@@ -169,7 +169,7 @@ _glamor_create_getcolor_fs_source(ScreenPtr screen, int stops_count,
         "}\n";
 
     if (use_array) {
-        XNFasprintf(&gradient_fs,
+        asprintf(&gradient_fs,
```

Comment:
Should we check the return value here too for consistency?

Reply:
indeed.